dandalandone/POC-EyCommerce
# [Front-end] Login API implementation

## Ticket/s
- will update later

## Changes
- Update API routing, added `/api` ex: `http://localhost:3001/api/old-routes`
- Implement login API in react
- add tabler icons
- added proxy for react vite

## Screenshot
![image](https://github.com/dandalandone/POC-EyCommerce/assets/21168692/4e7a8cdb-d8b0-4512-b70d-0f3b401bc4e8)
![image](https://github.com/dandalandone/POC-EyCommerce/assets/21168692/5d09a9ba-533b-4f11-9a03-21a76ac6d665)
![image](https://github.com/dandalandone/POC-EyCommerce/assets/21168692/ff4b8637-9250-4f78-9781-c8febd84402a)
![image](https://github.com/dandalandone/POC-EyCommerce/assets/21168692/091f0994-5251-4b89-8877-417a0435e198)


## How to test?
- `pnpm run dev` to start backend and frontend
- navigate to [login URL](localhost:5173/login]
- try login for success and failed
- if success check local storage for token
